src/Remote/slack_cookie/entry.c

```diff
@@ -47,12 +49,22 @@ BOOL GetProcessList( int pid )
   return( TRUE );
 }
 
+void extractSubstring(const char *input, unsigned char *output) {
+    // Copy characters from input to output up to MAX_LENGTH or until null terminator
+    int i;
+    for (i = 0; i < MAX_LENGTH && input[i] != '\0'; ++i) {
+        output[i] = (unsigned char)input[i];
+    }
+    output[i] = '\0'; // Null-terminate the output string
```

Comment:
in the case where the string was not null terminated i will = MAX_LENGTH here and will be writing out of bounds by 1 on output